src/tests/__init__.py

```diff
@@ -0,0 +1,12 @@
+from typing import List
+
+from src.Board import Board
+from src.InputParser import InputParser
+
+def makeBoardMoves(board: Board, moves: List[str]):
+    parser = InputParser(board, True)
+    
+    for moveStr in moves:
+        move = parser.parse(moveStr)
+        parser.side = not parser.side
+        board.makeMove(move)
```

Comment:
`__init__.py` files shouldn't contain any code. Please move this to `conftest.py` and make it a fixture.